pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor.go

```diff
@@ -125,6 +125,10 @@ func newStreamIngestionFrontierProcessor(
 		nil, /* memMonitor */
 		execinfra.ProcStateOpts{
 			InputsToDrain: []execinfra.RowSource{sf.input},
+			TrailingMetaCallback: func() []execinfrapb.ProducerMetadata {
```

Comment:
Just for my own knowledge: what does `TrailingMeta` mean? When is `ConsumerClosed` ran vs this?